python/src/FunctionalChaosResult_doc.i.in

```diff
@@ -212,3 +223,186 @@ coefficientsHistory : 2-d sequence of float
+variables in the group :math:`\vect{u}` and *only* them.
+For any :math:`\vect{\alpha} \in \mathcal{J}_{\vect{u}}^{\operatorname{ce}}`, let 
+:math:`\psi_{\vect{\alpha}}^{\operatorname{ce}}` be the orthogonal polynomial defined by :
+
+.. math::
+
+    \psi_{\vect{\alpha}}^{\operatorname{ce}}(\standardInputObservation_{\vect{u}})
+    = 
+    \begin{cases}
+    \prod_{\substack{i = 1 \\ i \in \vect{u}}}^\physicalInputDimension \pi_{\alpha_i}^{(i)} (z_i) & \textrm{if } \alpha_i = 0 \textrm{ for any } i \not \in \vect{u},. \\
+    1 & \textrm{if } \alpha_i = 0 \textrm{ for } i = 1, ..., \physicalInputDimension, \\
+    0 & \textrm{otherwise}.
+    \end{cases}
+
+Therefore :
+
+.. math::
+
+    \widetilde{h}_{\vect{u}}^{\operatorname{ce}}(\standardInputObservation_{\vect{u}})
+    = \sum_{\vect{\alpha} \in \mathcal{J}_{\vect{u}}^{\operatorname{ce}}}
+    a_{\vect{\alpha}} \psi_{\vect{\alpha}}^{\operatorname{ce}}(\standardInputObservation_{\vect{u}})
+
+for any :math:`\standardInputObservation_{\vect{u}} \in \standardInputSpace_{\vect{u}}`.
+Finally, the conditional expectation of the surrogate model is:
+
+.. math::
+  
+  \surrogateModel_{\vect{u}}\left(\physicalInputObservation_{\vect{u}}\right)
+  = \widetilde{h}_{\vect{u}}^{\operatorname{ce}}(T_{\vect{u}}\left(\physicalInputObservation_{\vect{u}}\right))
+
+where :math:`\standardInputRandomVector_{\vect{u}} = T_{\vect{u}}(\physicalInputRandomVector_{\vect{u}})`
+is the corresponding marginal mapping of the iso-probabilistic mapping
+:math:`\standardInputRandomVector = T(\physicalInputRandomVector)`.
+
+Parameters
+----------
+conditioningIndices : sequence of int in [0, inputDimension - 1]
+    The indices :math:`\vect{u}` of the input random vector to condition.
+
+Returns
+-------
+conditionalPCE : :class:`~openturns.FunctionalChaosResult`
+    The functional chaos result of the conditional expectation.
+    Its input dimension is :math:`\operatorname{card}(\vect{u})` and its output dimension
+    is :math:`\physicalOutputDimension` (i.e. the output dimension is unchanged)."
+    
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::FunctionalChaosResult::drawSelectionHistory
+"Draw the basis selection history.
+
+This is only available with :class:`~openturns.LARS`, and when the output dimension is 1.
+
+Returns
+-------
+graph : :class:`~openturns.Graph`
+    The evolution of the basis coefficients at each selection iteration"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::FunctionalChaosResult::getCoefficientsHistory
+"The coefficients values selection history accessor.
+
+This is only available with :class:`~openturns.LARS`, and when the output dimension is 1.
+
+Returns
+-------
+coefficientsHistory : 2-d sequence of float
+    The coefficients values selection history, for each iteration.
+    Each inner list gives the coefficients values of the basis terms at i-th iteration."
```

Comment:
That cannot be "the history of the coefficients" because there is some selection involved. I suggest "the history of the selected coefficients". Does it fit?

Reply:
Perfect!